runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -390,4 +390,6 @@
 	 */
 	@ScriptArguments(integer = 1)
 	public static final int NOTIFICATION_DELAY = 3347;
+
+	public static final int GROUP_STORAGE_ITEM_CONTAINER = 5269;
```

Comment:
This should include a `@ScriptArguments` annotation. Also, how did you come up with the name for this definition--it sounds more like a noun than a verb, and is rather confusing. If you wanted to match the bank building script, it could be named something like `GROUP_IRONMAN_STORAGE_BUILD`.